packages/@tailwindcss-postcss/src/index.ts

```diff
@@ -38,7 +42,13 @@ function tailwindcss(opts: PluginOptions = {}): AcceptedPlugin {
 
         function replaceCss(css: string) {
           root.removeAll()
-          root.append(postcss.parse(optimizeCss(css), result.opts))
+          let output = css
+          if (optimize) {
+            output = optimizeCss(output, {
+              minify: typeof optimize === 'object' ? optimize.minify : false,
```

Comment:
But if opts.optimize is true, minify should be true right? But in this case it would be set to false

Reply:
No there are 2 things happening:

1. "optimize" means, "optimize the output by handling nesting, merging adjacent at-rules, lowering CSS property values (`white` -> `#fff`) and applying vendor prefixes
2. Then there is also the `minify` part that minifies the CSS (mainly removing whitespacing). If minification happens, then optimization will also happen because that's done automatically by Lightning CSS.

Reply:
According to this example it says that `optimize: true` will minify too

```
module.exports = {
  plugins: {
    '@tailwindcss/postcss': {
      // This will optimize and minify the CSS output.
      optimize: true
    }
  }
}
```

Reply:
Aha you are right, I was wrong. `{ optimize: true }` should also minify by default! Opened a PR here to fix this: https://github.com/tailwindlabs/tailwindcss/pull/13201

Thanks for catching this!